src/global/ApolloClient/ApolloClient.ts

```diff
@@ -1,11 +1,19 @@
 import ApolloClient from 'apollo-boost';
+import { merge } from 'lodash';
+import { personaDefaults, personaResolvers } from 'global/ApolloLinkState/persona';
 
 //based on: https://www.apollographql.com/docs/react/networking/authentication/
 // https://dev.to/rdegges/please-stop-using-local-storage-1i04
 
 export const PS_TOKEN_NAME = 'psToken';
 
 export const client = new ApolloClient({
+  clientState: {
```

Comment:
Maybe it will be a good idea to create index.ts file for storing whole client state like 
```
import { withClientState } from 'apollo-link-state';
import _ from 'lodash';

import filterSettings from './filterSettings';
import globalAlert from './globalAlert';

export default globalCache =>
  withClientState({
    ..._.merge(filterSettings, globalAlert),
    cache: globalCache,
  });
```
And keep defaults and resolvers in one single object because they won't be used separately.

Reply:
but we dont use apollo-link-state this way

Reply:
Please, see:
[https://www.apollographql.com/docs/link/links/state/#with-apollo-boost](https://www.apollographql.com/docs/link/links/state/#with-apollo-boost)

Reply:
In that link you have provided me, in a section `Organizing your resolvers`, there is described what I wanted to achieve:
https://www.apollographql.com/docs/link/links/state/#organizing-your-resolvers
But this is not a big deal for me, we can import and export defaults and resolver if you prefer that way.